stan/math/prim/fun/eigenvalues.hpp

```diff
@@ -17,7 +17,7 @@ namespace math {
  */
 template <typename EigMat, require_eigen_matrix_dynamic_t<EigMat>* = nullptr,
           require_not_vt_complex<EigMat>* = nullptr>
-inline Eigen::Matrix<complex_return_t<value_type_t<EigMat>>, 1, -1> eigenvalues(
+inline Eigen::Matrix<complex_return_t<value_type_t<EigMat>>, -1, 1> eigenvalues(
```

Comment:
The change looks good, but before merging, I'd like to look at the types and tests for the function to see how this wasn't trapped prior to this merge.